ocp_resources/virtual_machine_clone.py

```diff
@@ -0,0 +1,82 @@
+from ocp_resources.constants import TIMEOUT_4MINUTES
```

Comment:
link to the api is missing

Reply:
currently no docs about this API in ocp 4.14, added TODO comment